src/gql/queries/get-patch-tasks.ts

```diff
@@ -17,7 +16,7 @@ export const GET_PATCH_TASKS = gql`
   ) {
     patchTasks(
       patchId: $patchId
-      limit: ${PATCH_TASKS_LIMIT}
+      limit: 10
```

Comment:
Can no longer embed variables in queries dynamically we must either hard code them or pass them in as a query param